app/models/family.rb

```diff
@@ -30,38 +47,15 @@ class Family < ActiveRecord::Base
   has_paper_trail
 
   before_validation :assign_family_type, if: [:new_record?, :brc?]
+  before_validation :assign_status
 
   validates :family_type, presence: true, inclusion: { in: TYPES }
-  validates :code, uniqueness: { case_sensitive: false }, if: 'code.present?'
-  validates :status, presence: true, inclusion: { in: STATUSES }
-  validate :client_must_only_belong_to_a_family
```

Comment:
@tvcam is this validation not needed anymore bro?

Reply:
Some validations are removed to satisfied the current feature, they will be reviewed again after before finalizing the PR @PirunSeng

Reply:
@PirunSeng I've fixed/updated related specs, but travi-ci is out of quota, and cannot do more build

Reply:
> 
> 
> Some validations are removed to satisfied the current feature, they will be reviewed again after before finalizing the PR @PirunSeng

Noted bro, I'm just afraid you removed it accidentally :). So, maybe you want to comment out or remove the method too.